Expand Thing Behavior section (#12)
namib-project/draft-coap-mud: draft-romann-mud-constrained.md

## Thing Behavior

### Exposing MUD URLs

Things MAY expose their MUD URL using a dedicated resource hosted under
`/.well-known/mud-url`.
If a MUD URL is exposed this way, the resource MUST offer at least one of the
specified serialization methods and MUST allow clients to choose between them
using the Accept option, if provided.

> If the Thing supports it, MUD URL resources SHOULD be advertised under
> `/.well-known/core` using the CoRE Link-Format {{!RFC6690}}, indicating the
> available Content-Formats using the `ct` parameter.

Reviewer: Maybe clarify what you mean by "it"

```suggestion
If the Thing supports resource discovery using a `/.well-known/core` resource, 
MUD URL resources SHOULD be advertised there using the CoRE Link-Format 
{{!RFC6690}}, indicating the available Content-Formats using the `ct` parameter.
```
